### PR TITLE
update interface `ISummarizerInternalsProvider.refreshLatestSummaryAck`

### PR DESCRIPTION
**Pull request by NicholasCouri**
update interface `ISummarizerInternalsProvider.refreshLatestSummaryAck`  to accept `IRefreshSummaryAckOptions` property instead of the previous arguments.
```diff
    async refreshLatestSummaryAck(
-       proposalHandle: string | undefined,
-       ackHandle: string,
-       summaryRefSeq: number,
-       summaryLogger: ITelemetryLogger,
+       options: IRefreshSummaryAckOptions,
    ):

## Does this introduce a breaking change?
yes

### CODE REVIEW

**Review by tyler-cai-microsoft — APPROVED** [review]

File: api-report/container-runtime.api.md [hunk 1]
```diff
@@ -448,6 +448,14 @@ export interface IProvideSummarizer {
     readonly ISummarizer: ISummarizer;
 }
 
+// @public
+export interface IRefreshSummaryAckOptions {
+    readonly ackHandle: string;
+    readonly proposalHandle: string | undefined;
+    readonly summaryLogger: ITelemetryLogger;
```
Comment (jatgarg) [inline api-report/container-runtime.api.md:455]: I don't think logger should be part of this.

File: BREAKING.md [hunk 2]
```diff
@@ -36,12 +38,25 @@ This field has been deprecated and will be removed in a future breaking change.
         fileName: string,
 -       createShareLinkType?: ShareLinkTypes,
 +       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
```
Comment (andre4i) [inline BREAKING.md:40]: why was this changed?

File: packages/runtime/container-runtime/src/summarizerTypes.ts [hunk 3]
```diff
@@ -114,6 +109,20 @@ export interface ISummarizeOptions {
...
+/**
+ * Data required to update internal tracking state after receiving a Summary Ack.
+ */
+ export interface IRefreshSummaryAckOptions {
+    /** Handle from the ack's summary op. */
+    readonly proposalHandle: string | undefined;
+    /** Handle from the summary ack just received */
+    readonly ackHandle: string;
+    /** Reference sequence number from the ack's summary op */
+    readonly summaryRefSeq: number;
```
Comment (tyler-cai-microsoft) [inline packages/runtime/container-runtime/src/summarizerTypes.ts:121]: Nit: `summaryRefSeq` => `summaryRefSeqNum` it may be a little easier to read. I didn't immediately think of reference sequence "number", granted I had enough context to figure it out.

**Review by andre4i — APPROVED** [review]

File: api-report/container-runtime.api.md (hunk 1 above)
Comment (NicholasCouri) [reply api-report/container-runtime.api.md:455]: yes, I considered that but the other interfaces do add the logger....

File: packages/runtime/container-runtime/src/summarizerTypes.ts (hunk 3 above)
Comment (NicholasCouri) [reply packages/runtime/container-runtime/src/summarizerTypes.ts:121]: I considered doing it but perhaps I will do later as the summaryRefSeq is being used on other public methods and I would need to change those as well.

File: BREAKING.md (hunk 2 above)
Comment (NicholasCouri) [reply BREAKING.md:40]: probably because I need to sync to the latest 0 will confirm.

**Review by kian-thompson — APPROVED** [review]

File: BREAKING.md (hunk 2 above)
Comment (NicholasCouri) [reply BREAKING.md:40]: Oh I think it is a bug on the  git tool
